Libraries/TestHelpers.psm1

```diff
@@ -795,12 +795,18 @@ Function Get-SSHKey ($XMLSecretFile) {
 		if ($privateSSHKey) {
 			$sshKeyPath = $privateSSHKey
 		}
+		$headerAuthInfo = $xmlSecret.secrets.RequestHeader
 	}
 	if ($privateSSHKey) {
 		if ($privateSSHKey -match "^(http|https)://") {
```

Comment:
Here should always ensure we are using htttps,  if detect htttp, throw exception, or stop processing and log Error messages.

Reply:
is that caused by http link is unsafe?

Reply:
yes, otherwise, the Header values in Map will be easily captured and parsed by a simple capturing tool.